md/pages.md

```diff
@@ -59,7 +59,7 @@ resources=[GET|identity/user, GET|identity/personalcontactdata, GET|identity/pro
 
 ## Custom page examples
 
-Two example custom pages are available in the portal. Both examples show how to:
+Two example custom pages are available in the Bonita BPM Portal. Both examples show how to:
```

Comment:
Rather "Two custom page examples" or "Two examples of custom page"
"the portal"